<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSIS</class>
 <widget class="QMainWindow" name="SSIS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(34, 40, 49)</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>581</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(34, 40, 49)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="studentTab">
     <attribute name="title">
      <string>STUDENT</string>
     </attribute>
     <widget class="QLineEdit" name="studentLineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>741</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 238)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButtonStudent">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 173, 181);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>DELETE</string>
      </property>
     </widget>
     <widget class="QTableView" name="studentTable">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>551</width>
        <height>411</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 238)</string>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="addButtonStudent">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 173, 181);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editButtonStudent">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 173, 181);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>EDIT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButtonStudent">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 173, 181);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SEARCH</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="courseTab">
     <attribute name="title">
      <string>COURSE</string>
     </attribute>
     <widget class="QPushButton" name="searchButtonCourse_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 173, 181);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SEARCH</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="courseLineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>741</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 238)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="addButtonCourse">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 173, 181);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editButtonCourse">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 173, 181);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>EDIT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButtonCourse">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 173, 181);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>DELETE</string>
      </property>
     </widget>
     <widget class="QTableView" name="courseTable">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>551</width>
        <height>411</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 238)</string>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
